test/e2e/cli/cli_test.go

```diff
@@ -1055,6 +1150,17 @@ func checkDevicesInFleetStatus(harness *e2e.Harness, fleetDevicesCount map[strin
 	return notMatched, err
 }
 
+// Checking the devicesSummary of fleets
+func checkDevicesSummary(harness *e2e.Harness) ([]string, error) {
+	out, err := harness.CLI("get", "fleet", "-s", "-o", "yaml")
```

Comment:
I think there is already a func for this in util or harness, please check and use.